app/controllers/admins/calculators_controller.rb

```diff
@@ -14,7 +14,11 @@ def index
     end
 
     def show
-      @calculator = Calculator.friendly.find(params[:id])
+      # show implementation add here
```

Comment:
don't use comments here, just write it in one string
```
def show; end
```

Reply:
Returned the previous implementation in one string